Source/console.cpp

```diff
@@ -32,7 +32,7 @@ Command::Command()
 				juce::File file{ arg.resolveAsFile() };
 				if (file.isDirectory())
 				{
-					for (const auto& child : file.findChildFiles(juce::File::TypesOfFileToFind::findFiles, true, "*.wav;*.flac"))
+					for (const auto& child : file.findChildFiles(juce::File::TypesOfFileToFind::findFiles, true, "*.wav;*.flac;*.aif;*.aiff;*.mp3"))
```

Comment:
Unfortunately, it's not quite that simple. JUCE has experimental support for .mp3, and if you take a look at the screenshot you posted in the PR, you'll notice that the data is actually incorrect. It says sample 1 but 2.083 seconds, which is pretty much impossible given conventional sample rates lol. 

zerochecker won't support those formats for the time being.

Reply:
I see, .mp3 indeed seems to be broken, .aiff files are working fine but I will leave those for your discretion.
The /Builds/MacOSX should be removed now.
First PR, thanks for your patience :)

Reply:
Ah thanks for checking .aiff! I'll make a note to add it and try it out in v0.0.2